icd/api/include/pipeline_binary_cache.h

```diff
@@ -152,7 +171,32 @@ class PipelineBinaryCache
 
     void FreePipelineBinary(const void* pPipelineBinary);
 
-    void Destroy() { this->~PipelineBinaryCache(); }
+    void* AllocMem(size_t memSize,
+                   size_t memAlignment = VK_DEFAULT_MEM_ALIGN,
+                   VkSystemAllocationScope allocationScope = VK_SYSTEM_ALLOCATION_SCOPE_OBJECT) const
```

Comment:
Don't use default arguments.  I'd prefer that you have a separate AllocMem() wrapper that passes the extra arguments.  I don't see a call to AllocMem() that takes more than the size so I think you can just remove the arguments on use the values on the call to pfnAllocation.